packages/office-ui-fabric-react/src/components/Panel/Panel.types.ts

```diff
@@ -214,86 +214,82 @@ export interface IPanelHeaderRenderer extends IRenderFunction<IPanelProps> {
+   * - Medium (480-639):  adapts to `PanelType.smallFixedFar` at this breakpoint
+   * - Large (640-1023): adapts to `PanelType.medium` at this breakpoint
+   * - XLarge (1024-1365): 48px fixed left margin, fluid width, 32px Left/Right padding
+   * - XXLarge (1366-up): 428px fixed left margin, fluid width, 40px Left/Right padding
    */
   large = 4,
 
   /**
-   * Renders the panel in 'large' mode, anchored to the far side (right in LTR mode), and is fixed at XXX-Large breakpoint.
-   * Small: \<adapts to smallFluid\>
-   * Medium: \<adapts to smallFixedFar\>
-   * Large: \<adapts to medium\>
-   * XLarge: 48px fixed left margin, 32px Left/Right padding
-   * XXLarge: 48px fixed left margin, 32px Left/Right padding
-   * XXXLarge: 940px width, (no redlines for padding, assuming previous breakpoint)
+   * Renders the Panel in `large` size, anchored to the far side (right in LTR mode), with a fixed width at XX-Large breakpoint.
+   * - Small (320-479): adapts to `PanelType.smallFluid` at this breakpoint
+   * - Medium (480-639): adapts to `PanelType.smallFixedFar` at this breakpoint
+   * - Large (640-1023): adapts to `PanelType.medium` at this breakpoint
+   * - XLarge (1024-1365): 48px fixed left margin, fluid width, 32px Left/Right padding
+   * - XXLarge (1366-up): 940px width, 40px Left/Right padding
    */
   largeFixed = 5,
 
   /**
-   * Renders the panel in 'extra large' mode, anchored to the far side (right in LTR mode).
-   * Small: \<adapts to smallFluid\>
-   * Medium: \<adapts to smallFixedFar\>
-   * Large: \<adapts to medium\>
-   * XLarge: \<adapts to large\>
-   * XXLarge: 176px fixed left margin, 40px Left/Right padding
-   * XXXLarge: 176px fixed left margin, 40px Left/Right padding
+   * Renders the Panel in `extra large` size, anchored to the far side (right in LTR mode).
+   * - Small (320-479): adapts to `PanelType.smallFluid` at this breakpoint
+   * - Medium (480-639): adapts to `PanelType.smallFixedFar` at this breakpoint
+   * - Large (640-1023): adapts to `PanelType.medium` at this breakpoint
+   * - XLarge (1024-1365): adapts to `PanelType.large` at this breakpoint
+   * - XXLarge (1366-1919): 176px fixed left margin, fluid width, 40px Left/Right padding
+   * - XXXLarge (1920-up): 176px fixed left margin, fluid width, 40px Left/Right padding
    */
   extraLarge = 6,
 
   /**
-   * Renders the panel in 'custom' mode using customWidth, anchored to the far side (right in LTR mode).
-   * Small: \<adapts to smallFluid\>
-   * Medium: \<adapts to smallFixedFar\>
-   * Large: 48px fixed left margin, 32px Left/Right padding
-   * XLarge: 644px width, 32px Left/Right padding
-   * XXLarge: 643px width, 40px Left/Right padding
+   * Renders the Panel in `custom` size using customWidth, anchored to the far side (right in LTR mode).
+   * - Has a fixed width provided by the `customWidth` prop
+   * - When screen width reaches the customWidth value it will get the behavior of a fluid width Panel
```

Comment:
```suggestion
   * - When screen width reaches the `customWidth` value it will behave like a fluid width Panel taking up
   * 100% of the viewport width
```